pagination/pagetoken_test.go

```diff
@@ -34,7 +35,37 @@ func TestParseOffsetPageToken(t *testing.T) {
 		assert.NilError(t, err)
 		assert.Equal(t, int64(30), pageToken3.Offset)
 	})
-
+	t.Run("skip", func(t *testing.T) {
+		t.Run("handle empty token with skip", func(t *testing.T) {
+			request1 := &freightv1.ListSitesRequest{
+				Parent: "shippers/1",
+				Skip:   30,
+			}
+			pageToken1, err := ParsePageToken(request1)
+			assert.NilError(t, err)
+			assert.Equal(t, int64(30), pageToken1.Offset)
+		})
+		t.Run("handle existing token with another skip", func(t *testing.T) {
+			request1 := &freightv1.ListSitesRequest{
+				Parent:   "shippers/1",
+				Skip:     50,
+				PageSize: 0,
```

Comment:
Would also like to see a non zero `PageSize` here ideally

Reply:
Adding another test.